jit.js

```diff
@@ -309,6 +502,15 @@ to single-step.
             return new Function("return function " + funcName + "(vm) {\n" + this.source.join("") + "\n}")();
         }
     },
+    deleteUnneededLabelBootstrap: function() {
```

Comment:
Please move this into `deleteUnneededLabels()`, there is no need to have a separate function. Also no need to have the `needsAnyLabel` property.